INFO:root:Endpoint http://localhost:3010/api/stats: False
HTTPConnectionPool(host='localhost', port=3010): Max retries exceeded with url: /api/stats (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001EF74907A10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
INFO:root:Endpoint http://localhost:3010/api/update-history: False
HTTPConnectionPool(host='localhost', port=3010): Max retries exceeded with url: /api/update-history (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001EF74A10190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
INFO:root:Endpoint http://localhost:3010/api/app-version: False
HTTPConnectionPool(host='localhost', port=3010): Max retries exceeded with url: /api/app-version (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001EF74A10A50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
INFO:root:Endpoint http://localhost:3010/api/changelog: False
HTTPConnectionPool(host='localhost', port=3010): Max retries exceeded with url: /api/changelog (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001EF74A14E90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
INFO:root:Notification manager: True

